README.md

```diff
@@ -1,56 +1,83 @@
 
 ### Key Scenarios
+
 These use cases aim to help with the following class of operations that sites would not want interrupted, even if the tab is occluded.
 
 * Downloading offline videos
 * Uploading large files
 * Video or image processing
 * Long payment transactions on shopping sites
+* Background geotracking or geofencing
 
 ## Non-goals
+
 * Extending the lifetime of a service worker when important work shouldn’t be interrupted by document closure is deferred to [Phase 2](/sw-explainer.md)
 * Extending the lifetime of a service worker when important work is triggered by a background event is deferred to [Phase 2](/sw-explainer.md)
 
-
 ## Creating a Progress Notification for a Task
-This explainer introduces a new `ProgressNotification` interface that can be initialized with a title and options to specify progress type. Browsers may use these inputs to surface details of the ongoing task in the UI when the site is not visible.
 
-The `update` method is for informing the browser when progress has been made, and to update the UI. This will take in a progress parameter ranging from [1-100] and `timeRemaining` value in seconds.
+This explainer introduces a new `ProgressNotification` interface that can be initialized with a `title` and options to specify a progress `type` (`'determinate'` or `'indeterminate'`). Browsers may use these inputs to surface details of the ongoing task in the UI when the site is not visible.
+
+The `update()` method is for informing the browser when progress has been made, and to update the UI. This will take in a progress parameter ranging from [1-100] and `timeRemaining` value in seconds.
 
-The `close` method is for informing the browser when a task has completed, and can be used by the browser to hide or update the UI as complete. 
+The `close()` method is for informing the browser when a task has completed, and can be used by the browser to hide or update the UI as complete. 
+
+### Determinate progress example
 
 ```js
-const progress = new ProgressNotification("Processing image...",
-                                          { type: 'determinate' });
+const progress = new ProgressNotification('Processing image', {
+  type: 'determinate',
+});
+
 while (!task.complete()) {
   task.step();
   progress.update(task.calculateProgress());
 }
+
 progress.close();
 ```
 
-## WebIDL
+### Indeterminate progress example
+
 ```js
+let id = navigator.geolocation.watchPosition(success, error, options);
+
+const progress = new ProgressNotification('Tracking geolocation', {
+  type: 'indeterminate',
+});
+
+button.addEventListener('click', () => {
+  navigator.geolocation.clearWatch(id);
```

Comment:
```suggestion
  navigator.geolocation.clearWatch(id);
  progress.close();
```

Also close progress UI in indeterminate example. (otherwise including the stop event listener isn't relevant, right?)

Reply:
Oh. Absolutely. Oversight of mine. Thank you!